src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -343,7 +349,7 @@ public void run() {
                         BkBasicTest.HOST,
                         "Connection: Close",
                         "",
-                        "Hello World!"
+                        BkBasicTest.HELLO_WORLD
```

Comment:
@dalifreire It looks like we can change this string to something else that is specific to this test